nestkernel/nest_time.cpp

```diff
@@ -162,7 +165,7 @@ Time::fromstamp( Time::ms_stamp t )
   // intended ones.
   tic_t n = static_cast< tic_t >( t.t * Range::TICS_PER_MS );
   n -= ( n % Range::TICS_PER_STEP );
-  long s = n / Range::TICS_PER_STEP;
+  long s = n * Range::TICS_PER_STEP_INV;
   double ms = s * Range::MS_PER_STEP;
```

Comment:
@heplesser @jakobj @gtrensch Agreed with guido -- I suggested later on that we make another PR with the changes in Time that aren't absolutely connected with the caching in Events or inverting the div operations.